boards/arm/sparkfun_pro_mirco_rp2040/sparkfun_pro_micro_rp2040.dts

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2021 Yonatan Schachter
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+
+#include <rpi_pico/rp2040.dtsi>
+#include "sparkfun_pro_micro_rp2040-pinctrl.dtsi"
+#include "sparkfun_pro_micro.dtsi"
+#include <freq.h>
+
+/ {
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,console = &uart0;
+		zephyr,code-partition = &code_partition;
+	};
+
+	aliases {
+		watchdog0 = &wdt0;
+	};
+
+	xtal_clk: xtal-clk {
+		compatible = "fixed-clock";
+		clock-frequency = <12000000>;
+		#clock-cells = <0>;
+	};
+};
+
+&flash0 {
```

Comment:
Now with the flash driver merged, this should be updated. (side note: it should also be fixed for the KB2040)

Reply:
Do you mean adding the chosen flash-controller? That seemed to be the only difference, which I fixed.